### DISCUSSION TITLE
How to use ArgoCD to install Helm chart in the Helm way so that they still show in 'helm list'?

### DISCUSSION THREAD

**Discussion by SebTardif** [argoproj/argo-cd#15676] [Q&A]
Helm is there to stay in K8s, and it's pretty nice to be able to see what a cluster have and easily get the back the manifest that was installed by using 'helm list'. I understand that ArgoCD is more about a bunch of manifest, and by default will just extract locally the manifest generated, then work with that. Do we have a way to use ArgoCD so that instead it uses helm the way it should be to install/update/delete?

Like ArgoCD could detect unsync state by comparing the Helm values and version.

**Comment by FernandoMiguel** [comment]
inst that what `Applications` are?
the same representation of argo applications like helm installs?
